src/components/ChatView.vue

```diff
@@ -44,7 +44,7 @@
 		</transition>
 		<MessagesList
 			:token="token" />
-		<NewMessageForm />
+		<NewMessageForm v-if="!isReadOnly" />
```

Comment:
I think @jancborchardt always wanted to keep the input form with a message that the conversation is locked? Or am I mixing things up now?

Reply:
I tried it at first but was more complicated as it requires adding a disabled property to many elements, so I removed it.
Also, most of the time the bottom-most message in the converstion is "admin has locked conversation" or so, so it's already clear that it's locked (and no one can post after this message)

@jancborchardt thoughts ?

Reply:
this is how it looks currently, please note the "you locked the conversation" message:

<img width="954" alt="image" src="https://user-images.githubusercontent.com/277525/95841278-76aebe00-0d45-11eb-8871-52806c32f52f.png">

@jancborchardt let me know if this is fine, otherwise I can look into adding a disabled field

Reply:
as discussed in chat, we'll go for the following approach when locked:

- hide all buttons around the message form
- display a disabled input field with the text "This conversation has been locked"

Reply:
Ref https://github.blog/2014-06-09-locking-conversations/